features/object.yml

```
@@ -1,10 +1,9 @@
-draft_date: 2024-06-05
 name: <object>
-description: The `<object>` element TODO.
-spec:
-  - https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element
-  - https://html.spec.whatwg.org/multipage/obsolete.html#param
+description: The `<object>` element represents an external resource such as an external plugin or nested browsing context.
```

"browsing context" is spec language that I wouldn't expect most web developers to be able to parse. Avoiding that and elaborating on the real reason this element exists:

```suggestion
description: The `<object>` element embeds an external resource such as a PDF or SVG document. It was historically used for plugins such as Shockwave Flash.
```

@ddbeck can you review so we don't have more rounds of description review after this?

---

Also consider that the description should work for `<embed>`. The word "embeds" would make it a bit circular, but I can't come up with anything better. Back to "represents"?

---

Building on @foolip's description, I'd make one minor tweak to avoid the word _embed_.

> The `<object>` element inserts an external resource such as a PDF or SVG file into the document. It was historically used for plugins such as Shockwave Flash.

---

I like @foolip's original, though `<embed>` will need to say "represents." It's not perfectly consistent, but produces a good outcome: you search for "embed" and you'll get both features.

---

> produces a good outcome: you search for "embed" and you'll get both features

I'm unsure why would that be a good outcome? My expectation of searching for "embed" would only return the `<embed>` element.

---

I think because they're closely related and easily confused (at least they are for me)—if I searched for one I'd appreciate being reminded of the other. But I don't have my heart set on this either; "represents" is fine too.

---

OK, I've updated the description to keep represents and use the rest of @foolip's description.